packages/babel-core/src/transformation/file/options/build-config-chain.js

```diff
@@ -33,7 +33,7 @@ export default function buildConfigChain(opts: Object = {}) {
   builder.mergeConfig({
     options: opts,
     alias: "base",
-    dirname: filename && path.dirname(filename),
```

Comment:
Looks like we default to this already as well https://github.com/loganfsmyth/babel/blob/386f0d2f6d148fea02ce83c1c903d2c9958024fa/packages/babel-core/src/transformation/file/options/build-config-chain.js#L176